_posts/2016-04-15-initial-thoughts-after-going-from-angular-to-react.md

```diff
@@ -0,0 +1,93 @@
+---
+layout: post
+title: "Initial Thoughts After Going From angular 1 to react-redux"
```

Comment:
From the trenches: What we learned building a production React Redux application

Tror vi med fordel kan droppe angular delene.